stripe/src/main/java/com/stripe/android/view/PaymentAuthWebView.kt

```diff
@@ -1,26 +1,10 @@
 package com.stripe.android.view
 
```

Comment:
What is the difference between PaymentAuthWebView and PaymentAutheWebViewClient?

Reply:
One is a [WebView](https://developer.android.com/reference/android/webkit/WebView), the other is a [WebViewClient](https://developer.android.com/reference/android/webkit/WebViewClient)